pkgs/development/tools/react-native-debugger/default.nix

```diff
@@ -1,5 +1,6 @@
-{ lib, stdenv, fetchurl, unzip, cairo, xorg, gdk-pixbuf, fontconfig, pango, gnome, atk, at-spi2-atk, at-spi2-core
-, gtk3, glib, freetype, dbus, nss, nspr, alsa-lib, cups, expat, udev, makeDesktopItem
+{ lib , stdenv , fetchurl , unzip , cairo , xorg , gdk-pixbuf , fontconfig , pango , gnome , atk , at-spi2-atk , at-spi2-core
+, gtk3 , glib , freetype , dbus , nss , nspr , alsa-lib , cups , expat , udev , makeDesktopItem , libdrm , libxkbcommon , mesa
+, makeWrapper
 }:
```

Comment:
We should format that with nixpkgs-fmt to prevent reflows which are hard to review

Reply:
Thanks for the review @SuperSandro2000 !